active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,146 @@
+Breadcrumb control
+===
+
+# Background
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+This is not the 'history' of the user's navigation, but the direct path of pages or folders from
+the root node to their current position. 
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 Breadcrumb no chevrons are interactable to show children.
+V2 Breadcrumb will add functionality to have flyouts from chevrons for every node,
+to view children of that node. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb
+
+Represents a control that shows a list of items, typically the navigation trail to the current location.
+
+
+### Examples
+
+The following example code creates a simple Breadcrumb control with three nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+
+XAML
+```XAML
+<Breadcrumb x:Name="Breadcrumb1" /> 
+```
+
+C#
+```cs
+Breadcrumb1.ItemsSource = new string[]{ "Node", "Node" , "Node"}
+```
+
+When there are too many items to display, a Breadcrumb "crumbles" and shows only the 
+last items.
+
+![Breadcrumb crumbling with 3 visible nodes](images/Breadcrumb_crumbling.PNG)
+
+
+If the last item in a Breadcrumb is too long to fit it will be truncated.
```

Comment:
Is it only the last item that gets truncated? What if a non-last item is really long, will it crumble away?

Reply:
I'll update this behavior explanation to more clearly reflect the hierarchy here. The current node is prioritized most. Crumbling as much as possible, and _then_ the last node truncates if needed. So for the scenario where any non-last node X is really long, if it was **too long for nodes X through the current leaf node to fit,** X would be crumbled.

Reply:
Thanks.

About the truncation, the picture here shows [TextTrimming.CharacterEllipsis](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.TextTrimming#fields), which works if it's text. If it's not text does it get clipped? Is text trimming specifically a special-case where the item is a string?

Reply:
discussion: If the last item in a Breadcrumb is text and is too long to fit it will be truncated. If not text this item will be clipped.